AppleScript/AppleScript.sublime-syntax

```diff
@@ -10,7 +10,7 @@ file_extensions:
 
 first_line_match: |-
   (?xi:
-    ^ \#! .* \bosascript\b                        # shebang
+    ^ \#! .* \bosascript\b(?!\s-l\sJavaScript)    # shebang
```

Comment:
I'd suggest to also accept multiple whitespace between parameters and maybe ensure JavaScript is terninated by a word boundary.

```suggestion
    ^ \#! .* \bosascript(?!\s+-l\s+JavaScript)\b  # shebang
```